src/compiler/tsc.ts

```diff
@@ -596,6 +601,18 @@ namespace ts {
             reportTimeStatistic("Total time", programTime + bindTime + checkTime + emitTime);
         }
 
+        const perfTotals = reduceProperties(perfTraces, (aggregate, value, key) => {

```

Comment:
this would probably be cleaner as a for-of loop anyway. It modifies the contents of `aggregate` and kind of hides the initialiser down at the bottom.

---
In reply to: [70891949](https://github.com/Microsoft/TypeScript/pull/9038#discussion_r70891949) [](ancestors = 70891949)